src/components/SpectroscopyForm.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from "react";
+import { Box, TextField, Typography } from "@mui/material";
+import NumberTextField from "./NumberTextField";
+import RunPlanButton from "./RunPlanButton";
+
+export type SpectroscopyFormData = {
+  numbers_of_points: number;
+  grid_size: number;
+  grid_origin_x: number;
+  grid_origin_y: number;
+  exposure_time: number;
+};
+
+function SpectroscopyForm() {
+  const [formData, setFormData] = useState<SpectroscopyFormData>({
+    numbers_of_points: 5,
+    grid_size: 5.0,
+    grid_origin_x: 0.0,
+    grid_origin_y: 0.0,
+    exposure_time: 0.1,
+  });
+  const [instrumentSession, setInstrumentSession] = useState("");
+
+  return (
+    <Box
+      sx={{
+        margin: 2,
+        display: "flex",
+        flexDirection: "column",
+      }}
+    >
+      <Typography
+        variant="h5"
+        component="h1"
+        sx={{ mb: 2, fontWeight: "bold" }}
+      >
+        spectroscopy_scan
```

Comment:
We don't need this if the Spectroscopy menu link drops us here. If you do want some text please not in camel case

Reply:
Removed